recipes/ipychart/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ipychart" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac01fbca69039e5631d1f2579edca80f47f15da3132df9aa7259cafd9e7e2846
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - ipychart
+
+about:
+  home: https://github.com/nicohlr/ipychart
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
```

Comment:
The `license_file` isn't in the source distribution. I've put in a PR to fix this in the next release:
https://github.com/nicohlr/ipychart/pull/1

For now, you'll need to manually package it:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

It would also be good to add a comment in the `meta.yaml` above the `license_file` line with the url to the PR just as a reminder to check if the source distribution includes the license file in future releases.